data.sql

```diff
@@ -5,4 +5,15 @@ VALUES
   ( 1, 'Agumon', '03-02-2020', 0, true, 10.23),
   ( 2, 'Gabumon', '15-11-2020', 0, true, 10.23),
   ( 3, 'Pikachu', '07-01-2020', 0, true, 10.23),
-  ( 4, 'Devimon', '12-05-2020', 0, true, 10.23);
```

Comment:
- [ ] Kindly ensure you input the correct records as required in the project requirements. Not all animals are `neutered` and the weight of some animals has `negative` values in the project requirement. It would also be a good idea to omit the `id` column as PostgreSQL will automatically generate and assign a unique value to the "id" column for each new row.

![image](https://github.com/sheezu900/vet-clinic/assets/98453979/4298d750-c8d0-4c9b-87c9-cb96a5281616)